tests/network-tests/src/flows/proposals/runtimeUpgradeProposal.ts

```diff
@@ -32,14 +35,15 @@ export default async function runtimeUpgradeProposal({ api, query, lock, env }:
   const buyMembershipFixture = new BuyMembershipHappyCaseFixture(api, query, [memberAcc])
   await new FixtureRunner(buyMembershipFixture).run()
   const [memberId] = buyMembershipFixture.getCreatedMembers()
+
   const createProposalsFixture = new CreateProposalsFixture(api, query, [
     {
       type: 'RuntimeUpgrade',
-      details: Utils.readRuntimeFromFile(runtimeUpgradeWasmPath),
+      details: (Utils.readRuntimeFromFile(runtimeUpgradeWasmPath) as unknown) as Bytes,
       asMember: memberId,
       title: 'To be cancelled by runtime',
       description: 'Proposal to be cancelled by runtime',
-    },
+    } as ProposalCreationParams<'RuntimeUpgrade'>,
```

Comment:
Avoid type assertions

Reply:
Addressed in [b552fba](https://github.com/Joystream/joystream/pull/3997/commits/b552fba6a40a75129fb358c4b771150313077472)